libnspr.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: libnspr
+  version: 4.35
+  epoch: 0
+  description: "Netscape Portable Runtime (NSPR) provides a platform-neutral API for system level and libc-like functions."
+  copyright:
+    - license: Mozilla Public License v2
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - mercurial
+      - bash
+
+var-transforms:
+  - from: ${{package.version}}
+    match: \.
+    replace: _
+    to: libnspr-package-version
+
+pipeline:
+  # we need to use runs because melange dont have a mercurial checkout pipeline yet
+  - runs: |
+      hg clone -r NSPR_${{vars.libnspr-package-version}}_RTM https://hg.mozilla.org/projects/nspr
+
+  - uses: autoconf/configure
+    with:
+      dir: nspr
+      opts: |
+        --enable-64bit \
+        --host=${{host.triplet.gnu}} \
+        --target=${{host.triplet.gnu}} \
+        --prefix=/usr
```

Comment:
Comparing with https://github.com/wolfi-dev/os/pull/7291/files we may be missing optimization flags and possibly some script cleanups that I found in Arch's packaging of this.